### PR TITLE
Enhance/merge commits

### PR DESCRIPTION
**Pull request by hanawa-suzuki**

### CODE REVIEW

File: codecov [hunk 1]
```diff
@@ -543,14 +543,7 @@ fi
 if [ "$commit_o" = "" ];
 then
   # merge commit -> actual commit
-  mc=$(git log -1 --pretty=%B 2>/dev/null | tr -d '[[:space:]]' || true)
```
Comment (stevepeak) [inline codecov]: We need to keep this section until we find a better solution server side.

Comment (brandon1780) [reply codecov]: Divide

Comment (brandon1780) [inline codecov]: Pull

Comment (brandon1780) [reply codecov]: VPN bower

**Comment by davisjackie321** [comment]
Sucks

**Comment by drazisil** [comment]
I don't understand what this PR would do. @hanawa-suzuki can you share more details if this is still something you want?

**Pull request closed without merging**